spec/base_spec_helper.rb

```diff
@@ -58,6 +59,11 @@
   # rspec-rails.
   config.infer_base_class_for_anonymous_controllers = false
 
+  # Show retries in test output
+  config.verbose_retry = true
+  # Set maximum retry count
+  config.default_retry_count = 0
```

Comment:
We don't need to set this. The default is `1` which means that the example is tried once. The name is confusing.

> Note that currently this is a 'try' count. If increased from the default of 1, all examples will be retried. We plan to fix this as a breaking change in version 1.0.

https://github.com/NoRedInk/rspec-retry#configuration